src/Functions/polyval3.m

```diff
@@ -1,11 +1,21 @@
-%% polyval3 - evaluate an (independent component) polynomial in R3
-%
-% Andreas Husch
-% Centre Hospitalier de Luxembourg, Dep. of Neurosurgery /
-% University of Luxembourg - Luxembourg Centre for Systems Biomedicne
-% 2014 - 2017
-% mail@andreashusch.de, husch.andreas@chl.lu
 function val = polyval3(polyCoeffs, evalAt)
+% evaluate an (independent component) polynomial in R3
+%
+% USAGE:
+%
+%    val = polyval3(polyCoeffs, evalAt)
+%
+% INPUTS: 
+%    polyCoeffs:    Coefficient matrix
+%    evalAt:        
```

Comment:
@adhusch Could you please provide a description of the input and the output?

Reply:
This method evaluates a function (precisely a polynomial mapping from R^1 -> R^3) i.e. sort of y=f(x) where x \in R^1 and y \in R^3 at the given point x="evalAT".